src/components/Loader/styled.tsx

```diff
@@ -29,8 +29,8 @@ export const LoadingRows = styled.div`
 `
 
 export const loadingOpacityMixin = css<{ $loading: boolean }>`
-  filter: ${({ $loading }) => ($loading ? 'grayscale(1)' : 'none')};
-  opacity: ${({ $loading }) => ($loading ? '0.4' : '1')};
+  filter: ${({ $loading }) => ($loading ? 'grayscale(1)' : 'none')} !important;
```

Comment:
why is `!important` necessary?

Reply:
Thought it was an issue with how the styles were being passed but looks like we dont need it.